MapleServer2/PacketHandlers/Game/StatPointHandler.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using System;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class StatPointHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.STAT_POINT;
+
+        public StatPointHandler(ILogger<StatPointHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            byte mode = packet.ReadByte();
+
+            switch (mode)
+            {
+                case 2: 
+                // Increment attribute point
+                    HandleStatIncrement(session, packet);
+                    // follow up SendStat packet to refresh updated stats
+                    break;
+                case 3:
+                // Reset attribute distribution
+                    HandleResetStatDistribtuion(session, packet);
+                    // follow up SendStat packet to refresh updated stats
+                    break;
+            }
+        }
+
+        private void HandleStatIncrement(GameSession session, PacketReader packet)
+        {
+            byte statTypeIndex = packet.ReadByte();
+            Console.WriteLine("statIndex value: " + statTypeIndex);
```

Comment:
Probably can remove this debug. Maybe we should add a check that disallows us to add to other stats (other than 0x00 to 0x04, 0x11).

Reply:
Debug removed! Will discuss adding a check.

Reply:
Will add a check in the next iteration of this feature. Will try to get the PR merged first :)